integrations/launchdarkly/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,88 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: projects
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"launchDarklyProject"'
+          properties:
+            id: ._id
+            tags: .tags
+            includeInSnippetByDefault: .includeInSnippetByDefault
+            defaultClientSideAvailability: .defaultClientSideAvailability
+            url: ("https://app.launchdarkly.com" + ._links.self.href | tostring)
+  - kind: flags
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"launchDarklyFeatureFlag"'
+          properties:
+            kind: .kind
+            description: .description
+            creationDate: '.creationDate |  (strptime("%Y-%m-%d %H:%M:%S") | strftime("%Y-%m-%dT%H:%M:%SZ"))'
+            includeInSnippet: .includeInSnippet
+            clientSideAvailability: .clientSideAvailability
+            temporary : .temporary
+            tags: .tags
+            maintainer: ._maintainer
+            environments: .environments
+            variations: .variations
+            customProperties: .customProperties
+            archived: .archived
+          relations:
+            project: .__projectKey   
+  - kind: auditlog
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: ._id
+          title: .titleVerb
+          blueprint: '"launchDarklyAuditLog"'
+          properties:
+            accountId: ._accountId
+            date: '.date |  (strptime("%Y-%m-%d %H:%M:%S") | strftime("%Y-%m-%dT%H:%M:%SZ"))'
+            kind: .kind
+            name: .name
+            description: .description
+            shortDescription: .shortDescription
+            member: .member
+            title: .title
+            accesses: .accesses
+            target: .target
+          relations:
+            project: 'select(.kind | IN("project")) | .accesses[] | .resource | split("/") | .[-1]'
+  - kind: environments

```

Comment:
rename kind to `environment` - consistent with the ones you defined in the spec.yaml